javascript-practice/src/js/views/loginView.js

```diff
@@ -0,0 +1,91 @@
+import CommonLoginRegisterView from './commonLoginRegisterView';
+import * as CONSTANT from '../constants/constant';
+
+export default class LoginView extends CommonLoginRegisterView {
+  constructor() {
+    super();
+
+    this.parentElement = document.querySelector('.form');
+    this.loginPage = document.getElementById('loginPage');
+  }
+
+  /**
+   * Get data from user input
+   * @returns {Object || null} Return object or null
+   */
+  getDataFromForm() {
+    const { loginForm } = document.forms;
```

Comment:
Is there any way to get form? This worked if you have one form only.

Reply:
Fixed.